tests/local/network.go

```diff
@@ -470,8 +470,9 @@ func (n *LocalNetwork) AddSubnetValidators(ctx context.Context, subnetID ids.ID,
 	)]
 
 	// consume some of the extraNodes
-	newValidatorNodes := n.extraNodes[0:count]
-	copy(n.extraNodes, n.extraNodes[count:])
+	var newValidatorNodes []*tmpnet.Node
```

Comment:
```suggestion
	newValidatorNodes := make([]*tmpnet.Node, len(n.extraNodes[0:count]))
```
We need to preallocate the destination, otherwise nothing will be copied. [Example](https://go.dev/play/p/bZ4utbW6N2l)

Reply:
The `append` approach you adopted also works.